src/EA.Weee.Tests.Core/DirectRegistrantHelper.cs

```diff
@@ -24,13 +25,23 @@ public static class DirectRegistrantHelper
             return Tuple.Create(complianceYear, country);
         }
 
-        public static Tuple<Domain.Organisation.Organisation, DirectRegistrant, Domain.Producer.RegisteredProducer> CreateOrganisationWithRegisteredProducer(DatabaseWrapper wrapper, string companyName, string prn, int complianceYear)
+        public static Tuple<Domain.Organisation.Organisation, DirectRegistrant, Domain.Producer.RegisteredProducer> CreateOrganisationWithRegisteredProducer(DatabaseWrapper wrapper, string companyName, string prn, int complianceYear, string companyNumber = null, Domain.Producer.AuthorisedRepresentative authorisedRepresentative = null)
         {
+            companyNumber = companyNumber ?? "123456789";
             var organisation = Domain.Organisation.Organisation.CreateRegisteredCompany(companyName, "123456789");
-            var address = new Domain.Organisation.Address("1", "street", "Woking", "Hampshire", "GU21 5EE", wrapper.WeeeContext.Countries.First(), "12345678", "test@co.uk");
+            var address = new Domain.Organisation.Address("primary 1", "street", "Woking", "Hampshire", "GU21 5EE", wrapper.WeeeContext.Countries.First(), "12345678", "test@co.uk");
             organisation.AddOrUpdateAddress(Domain.AddressType.RegisteredOrPPBAddress, address);
 
-            var directRegistrant = DirectRegistrant.CreateDirectRegistrant(organisation, null, null, null, null, null);
+            var contact = new Domain.Organisation.Contact("first name", "last name", "position");
+
+            Domain.Producer.AuthorisedRepresentative authRep = null;
```

Comment:
Maybe I am missing something but can't see what the purpose of creating a new var of AuthorisedRepresentative called authRep is here. Can't you pass "authorisedRepresentative" directly into CreateDirectRegistrant? The null check won't do anything, because if its null, authRep will still be null

Reply:
Thanks yup absolutely no point...

Reply:
Updated...